mcs_benchmark_data/models/submission.py

```diff
@@ -27,16 +27,17 @@ def to_rdf(self, *, graph: Graph) -> Resource:
 
         resource.add(SCHEMA.name, self._quote_rdf_literal(self.name))
         resource.add(SCHEMA.description, self._quote_rdf_literal(self.description))
-        resource.add(SCHEMA.dateCreated, self.date_created)
+        resource.add(SCHEMA.dateCreated, Literal(self.date_created))
         resource.add(SCHEMA.isBasedOn, self._quote_rdf_literal(self.is_based_on))
+
         for contributor in self.contributors:
             resource.add(MCS.contributor, self._quote_rdf_literal(contributor))
 
         resource.add(
             SCHEMA.softwareApplication, self._quote_rdf_literal(self.result_of[0])
         )
-        resource.add(SCHEMA.endTime, self.result_of[1])
-        resource.add(SCHEMA.startTime, self.result_of[2])
+        resource.add(SCHEMA.endTime, Literal(self.result_of[1]))
```

Comment:
I guess this is part of what changed in the data/ files? Probably should have been in another PR.